test/commands/devtool.test.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+'use strict';
+
+// const { test: oclifTest } = require("@oclif/test");
+const {stdout} = require("stdout-stderr");
+const assert = require("assert");
+const promisify = require('util').promisify;
+const sleep = promisify(setTimeout);
+const fetch = require('node-fetch');
+
+describe("devtool command", function() {
+    it("devtool starts and serves html", async function() {
+        // set up server
+        const DevToolCommand = require("../../src/commands/asset-compute/devtool");
+        stdout.start();
+        const devtool = new DevToolCommand([]);
+        devtool.run([]);
+        await sleep(50);
```

Comment:
50ms seems random... It may bot work for all machines, as the starting/running of devtool may take longer on slower machines. Turn it into a constant that is easily locatable, so if someone works on a slower machine, they may identify test failures related to timing faster.

Reply:
I think it's ok, but maybe give it a bit more time? We'll see in CI test executions I guess.

The one way to make it more stable would be to have a `startup()` API that you could await on. I.e. just the startup of the server... but not sure how much refactoring that would mean.

Reply:
Hmm, I think the `startup()` is a good idea for the future. For now, I'll just give it more buffer time since different OS may be slower.  I just changed it to 500ms